### PR TITLE
State icon #468 

### LINKED ISSUE #468: Create an interactive icon to manage widget status
Don't you think this very one could be useful as a general component in sw ?

could you also add a super simple test where you just init the component (so that codecov stops complaining)

_Originally posted by @12rambau in https://github.com/12rambau/sepal_ui/pull/439#discussion_r870454374_

### PR DESCRIPTION
**Pull request by dfguerrerom**
Aims to close #468

### CODE REVIEW

File: docs/source/widgets/stateicon.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+=========
+
+:code:`StateIcon` is a custom widget to provide easy to use a mutable icon state: displaying a short description of the state with a tooltip and a descriptive color. This widget is intended to be linked with a :code:`SepalModel` trait, which will be in charge of changing the widget state. Here is an example:
+
+.. jupyter-execute::
+    :raises:
+    
+    import ipyvuetify as v 
+    from traitlets import Unicode
+    from sepal_ui import sepalwidgets as sw
```
Comment (12rambau) [inline docs/source/widgets/stateicon.rst:11]: you forget to import `Model` that is use on line 18

File: sepal_ui/sepalwidgets/widget.py [hunk 2]
```diff
@@ -126,3 +127,62 @@ def _clip(self, widget, event, data):
...
+    def __init__(self, model, model_trait, states=None):
+
+        default_states = {
+            "valid": ("Valid", color.success),
+            "non_valid": ("Not valid", color.error),
+        }
+
+        self.states = default_states if not states else states
+
+        self.right = True
```
Comment (12rambau) [inline sepal_ui/sepalwidgets/widget.py]: what is the purpose of this attribute ? It's set to True at init level and then never changed

Comment (12rambau) [reply sepal_ui/sepalwidgets/widget.py]: https://github.com/12rambau/sepal_ui/pull/469/commits/4756607c08c4513b1e128e8c880d7231cb63c4cb

**Review by 12rambau — APPROVED** [review]

**Comment by dfguerrerom** [comment]
Closing because it's already requested in https://github.com/12rambau/sepal_ui/pull/439

**Pull request closed without merging**